packages/reporter/src/commands/commands.scss

```
@@ -322,6 +322,22 @@
     text-overflow: ellipsis;
   }
 
+  .command-message-options {
+    display: block;
+    white-space: initial;
+    font-size: 0.85em;
+    opacity: 0.9;
+
+    .object-viewer {
+      line-height: 1.2;
+      margin: 4px 8px 4px 0;
+    }
+
+    .option-item {
+      margin-left: 16px;
+    }
```

Fix placement of expand arrow
```suggestion
    }
    
    .collapsible-indicator {
      top: 1px;
      position: relative;
    }
```

---

Done.